src/strategies/offer_forwarder/OfferForwarder.sol

```diff
@@ -26,23 +26,18 @@ contract OfferForwarder is ILiquidityProvider, Forwarder {
   }
 
   /// @inheritdoc ILiquidityProvider
-  function newOffer(
-    IERC20 outbound_tkn,
-    IERC20 inbound_tkn,
-    uint wants,
-    uint gives,
-    uint gasreq,
-    uint gasprice, // keeping gasprice here in order to expose the same interface as `OfferMaker` contracts.
-    uint pivotId
-  ) external payable returns (uint offerId) {
-    gasprice; // ignoring gasprice that will be derived based on msg.value.
+  function newOffer(IERC20 outbound_tkn, IERC20 inbound_tkn, uint wants, uint gives, uint pivotId)
+    external
+    payable
+    returns (uint offerId)
+  {
     offerId = _newOffer(
       OfferArgs({
         outbound_tkn: outbound_tkn,
         inbound_tkn: inbound_tkn,
         wants: wants,
         gives: gives,
-        gasreq: gasreq,
+        gasreq: offerGasreq(),
         gasprice: 0,
```

Comment:
You use gasprice:0 to use mangrove's gasprice. You specify gasreq, but could use `type(uint24).max` to use Forwarder's/Direct's default (which is offerGasreq(). You decide :)